test/sanity/validate-modules/validate-modules

```diff
@@ -41,6 +41,14 @@ from schema import doc_schema, option_schema, metadata_schema
 from utils import CaptureStd, parse_yaml
 from voluptuous.humanize import humanize_error
 
+from ansible.compat.six import PY3, with_metaclass
+
+if PY3:
+    # Because there is no ast.TryExcept in Python 3 ast module
+    ast.TryExcept = ast.Try
+    # REPLACER_WINDOWS from ansible.executor.module_common is byte
+    # string but we need unicode for Python 3
+    REPLACER_WINDOWS = REPLACER_WINDOWS.decode('utf-8')
```

Comment:
You cannot simply change mode to `rb` because there are a lot of code parts where comparison with the unicode is used like `'sys.exit(' in self.text`, `('GNU General Public License' not in self.text and 'version 3' not in self.text)`, `'WANT_JSON' not in self.text` and more.

I think that replacing all these strings with bytes and load module file as bytes is a bad idea because module files are unicode and should be loaded as unicode.

Possibilities:
* Use my code and just convert REPLACER_WINDOWS to unicode
* Get rid of REPLACER_WINDOWS which is used only once a replace it directly with the unicode string `u'# POWERSHELL_COMMON'` which is IMHO worse and going against DRY principe.

Do you have a better idea?